command/sub8_missions/sub8_missions/ball_drop.py

```diff
@@ -60,17 +61,17 @@ def run(self, args):
      #   yield enable_service(SetBoolRequest(data=True))
 
         try:
-            vamp_txros = yield self.nh.get_service_client('/guess_location',
-                                                          GuessRequest)
-            ball_drop_req = yield vamp_txros(GuessRequestRequest(item='ball_drop'))
-            if ball_drop_req.found is False:
+            save_pois = rospy.ServiceProxy(
```

Comment:
Using `rospy` and `txros` at the same time is going to break things.

This is why I made the `TxPoiClient ` [used in navigator](https://github.com/uf-mil/NaviGator/blob/master/mission_control/navigator_missions/navigator_missions/navigator.py#L113)